### PR TITLE
ENH: add LA county

### PR DESCRIPTION
**Pull request by sglyon**
This class uses pyppeteer to scrape time series data for number of cases, deaths, positive tests, and negative tests from the LA county dashboard.

### CODE REVIEW

**Comment by sglyon** [comment]
Note that this requires python >= 3.7, which shouldn't be a problem as we use that version anyway!

File: src/cmdc/datasets/official/CA/counties/la.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+        # get cases info
+        await page.waitForXPath("//div[@id='myplot']/div")
+        cum_cases = await find_chart_data(page, "myplot", "cases_total", "cases")
+        cum_deaths = await find_chart_data(page, "myplot", "deaths_total", "deaths")
+
+        # move to testing page
+        await (await page.waitForXPath(r"//a[@data-value='testing']")).click()
+
+        # TODO: figure out how to click this...
```
Comment (cc7768) [inline src/cmdc/datasets/official/CA/counties/la.py]: Is this TODO important to resolve before merging?

**Review by cc7768 — APPROVED** [review]
As long as that TODO isn't blocking, I think this looks good.

File: src/cmdc/datasets/official/CA/counties/la.py (hunk 1 above)
Comment (sglyon) [reply src/cmdc/datasets/official/CA/counties/la.py]: oops sorry, changed plan. that should be deleted

**Pull request merged**